packages/fetchai/connections/http_server/tests/test_http_server.py

```diff
@@ -268,7 +269,7 @@ async def test_bad_performative_get_timeout_error(self):
     @pytest.mark.asyncio
     async def test_late_message_get_timeout_error(self):
         """Test send get request w/ 200 response."""
-        self.http_connection.channel.timeout_window = 1
+        self.http_connection.channel.timeout_window = 0.2
```

Comment:
can you explain? particularly why `a < b` now instead of `a > b` as it was

Reply:
just decreased timeout and next sleep. make it a bit faster. looks ridiculous, but many sleeps cosnumea lot of time.
it's better to get rid of all sleeps in tests.